geometry_msgs/TransformStamped e_frame
float32 height
float32 E
bool is_oblate
